build-logic/src/main/kotlin/dokkabuild/DokkaBuildProperties.kt

```diff
@@ -48,11 +49,46 @@ abstract class DokkaBuildProperties @Inject constructor(
     val kotlinLanguageLevel: Provider<KotlinVersion> =
         dokkaProperty("kotlinLanguageLevel", KotlinVersion::fromVersion)
 
+    val integrationTestParallelism: Provider<Int> =
+        dokkaProperty("integration_test.parallelism", String::toInt)
+            // fallback & warn if the old property is used
+            .orElse(
+                providers.gradleProperty("dokka_integration_test_parallelism")
+                    .map {
+                        logger.warn("dokka_integration_test_parallelism is deprecated - use org.jetbrains.dokka.integration_test.parallelism instead")
+                        it.toInt()
+                    }
+            )
+
+    val integrationTestExhaustive: Provider<Boolean> =
+        dokkaProperty("integration_test.exhaustive", String::toBoolean)
+            // fallback & warn if the old property is used
+            .orElse(
+                providers.gradleProperty("dokka_integration_test_is_exhaustive")
+                    .map {
+                        logger.warn("dokka_integration_test_is_exhaustive is deprecated - use org.jetbrains.dokka.integration_test.exhaustive instead")
+                        it.toBoolean()
+                    }
+            )
+            .orElse(
+                providers.environmentVariable("DOKKA_INTEGRATION_TEST_IS_EXHAUSTIVE")
+                    .map {
+                        logger.warn("DOKKA_INTEGRATION_TEST_IS_EXHAUSTIVE is deprecated - use ORG_GRADLE_PROJECT_org.jetbrains.dokka.integration_test.exhaustive instead")
+                        it.toBoolean()
+                    }
+            )
```

Comment:
JFYI this particular property (`dokka_integration_test_is_exhaustive)` can actually be dropped altogether. The versions of Kotlin/Gradle/etc that are run under it are not compatible (and haven't been for 2 years), so the tests fail under this flag. And I don't know if it makes sense to invest time into making it work now, so I'd just drop it and return to it when/if it's needed

Reply:
Can that be done in a later PR? :) I just want to focus on updating the Gradle config in this chain of PRs, and the `isExhaustive` variable is still used in code.